scripts/xtensa-build-zephyr.py

```diff
@@ -228,6 +207,11 @@ def parse_args():
 			    help="""Use an output subdirectory for each platform.
 Otherwise, all firmware files are installed in the same staging directory by default.""")
 
+	parser.add_argument("--no-interactive", default=False, action="store_true",
+			    help="""Run script in non-interactive mode when user input can not be provided.
+This should be used with programatic script invocations (eg. Continuous Integration).
```

Comment:
```suggestion
This should be used with programmatic script invocations (eg. Continuous Integration).
```
Last checkpatch warning and then you're checkpatch-clean